[2023-12-01T15:23:57.998+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_is_file_in_s3_available manual__2023-12-01T15:23:49.409530+00:00 [queued]>
[2023-12-01T15:23:58.006+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_is_file_in_s3_available manual__2023-12-01T15:23:49.409530+00:00 [queued]>
[2023-12-01T15:23:58.006+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-12-01T15:23:58.022+0000] {taskinstance.py:1382} INFO - Executing <Task(S3KeySensor): tsk_is_file_in_s3_available> on 2023-12-01 15:23:49.409530+00:00
[2023-12-01T15:23:58.026+0000] {standard_task_runner.py:57} INFO - Started process 36659 to run task
[2023-12-01T15:23:58.030+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'zillow_analytics_dag', 'tsk_is_file_in_s3_available', 'manual__2023-12-01T15:23:49.409530+00:00', '--job-id', '70', '--raw', '--subdir', '/home/ubuntu/airflow/dags', '--cfg-path', '/tmp/tmpjqc7k5sa']
[2023-12-01T15:23:58.031+0000] {standard_task_runner.py:85} INFO - Job 70: Subtask tsk_is_file_in_s3_available
[2023-12-01T15:23:58.068+0000] {task_command.py:416} INFO - Running <TaskInstance: zillow_analytics_dag.tsk_is_file_in_s3_available manual__2023-12-01T15:23:49.409530+00:00 [running]> on host ip-172-31-24-255.us-west-2.compute.internal
[2023-12-01T15:23:58.143+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='myemail@domain.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='zillow_analytics_dag' AIRFLOW_CTX_TASK_ID='tsk_is_file_in_s3_available' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T15:23:49.409530+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-01T15:23:49.409530+00:00'
[2023-12-01T15:23:58.144+0000] {s3.py:106} INFO - Poking for key : s3://myawsbucket-new/response_data_01122023152351.csv
[2023-12-01T15:23:58.149+0000] {base.py:73} INFO - Using connection ID 'aws_s3_conn' for task execution.
[2023-12-01T15:23:58.149+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='aws_s3_conn', conn_type='aws') credentials retrieved from login and password.
[2023-12-01T15:24:03.852+0000] {s3.py:106} INFO - Poking for key : s3://myawsbucket-new/response_data_01122023152351.csv
[2023-12-01T15:24:08.868+0000] {s3.py:106} INFO - Poking for key : s3://myawsbucket-new/response_data_01122023152351.csv
[2023-12-01T15:24:08.885+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-12-01T15:24:08.890+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=zillow_analytics_dag, task_id=tsk_is_file_in_s3_available, execution_date=20231201T152349, start_date=20231201T152357, end_date=20231201T152408
[2023-12-01T15:24:08.934+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-01T15:24:08.951+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
